modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2DeviceFlowTestCase.java

```diff
@@ -46,10 +47,7 @@
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
No `*` imports. Please update your IDE settings correctly. Check other places as well.

Reply:
Fixed with https://github.com/wso2/product-is/pull/15906/commits/38e6cdd0e3f7357405e8fe1891714dcfcb17624b